bin/unify_model_lengths.py

```diff
@@ -45,17 +45,13 @@ def parse_args():
     parser.add_argument(
         "-plh", "--peptide_max_len", help="Maximum length of peptide used for prediction", type=int, default=13
     )
-
     parser.add_argument(
-        "-o",
-        "--output",
+        "-log_s",
```

Comment:
these parameter names became quite long, maybe just "s" for the short form and "log_suffix" for the long form?